database/schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE DibsOnDinnerPhotos;
+
+USE DibsOnDinnerPhotos;
+
+CREATE TABLE photos (
+  id int NOT NULL AUTO_INCREMENT,
+  url text NOT NULL,
+  restaurant_id int NOT NULL,
+);
```

Comment:
Not to be nitpicky but there no newline at the end of the file :)

Reply:
other than that lgtm

Reply:
@Try-Try-Again The whole file is new!
It's actually not working. It creates the database, it seems like it only uses line 1.